src/rt/sections_win64.d

```diff
@@ -81,13 +72,19 @@ void initSections() nothrow @nogc
     debug(PRINTF) printf("found .data section: [%p,+%llx]\n", dataSection.ptr,
                          cast(ulong)dataSection.length);
 
-  version (LDC) {} else
-  {
     import rt.sections;
     conservative = !scanDataSegPrecisely();
-  }
 
-    version (conservative) // LDC: compile-time
+    version (LDC)
+    {
+        /* FIXME: Precise DATA/TLS GC scanning requires compiler support
+         * (emitting mutable pointers into special sections bracketed
+         * by _{D,T}P_{beg,end} symbols).
```

Comment:
Oh, I forgot this is not supported by LDC. It could be helpful for dmd or the dmdserver for Visual D that are built with LDC now.

Reply:
It would be interesting to have a figure in this regard. - Feel free to open an issue so that we don't forget, but I don't expect much of an impact for non-32-bit targets (I care about).